xblockutils/public/studio_container.js

```diff
@@ -1,20 +1,21 @@
 function StudioContainerXBlockWithNestedXBlocksMixin(runtime, element) {
     var $buttons = $(".add-xblock-component-button", element),
+        $view = $('.add-xblock-component', element),
```

Comment:
@eliangcs `$component` is a bit generic, but I'm having a hard time thinking of a better name myself. `$addComponent`, perhaps?